stix_shifter/stix_translation/src/utils/transformers.py

```diff
@@ -100,6 +100,16 @@ def transform(registryvalues):
         return converted_value
 
 
+class AwsToTimestamp(ValueTransformer):
+    """A value transformer to format UTC timestamp"""
```

Comment:
Could you add examples in the comment to show the AWS timestamp and the transformed UTC timestamp?

Reply:
We will add the comment

Reply:
done